WasatchNET/EEPROM.cs

```diff
@@ -282,6 +280,14 @@ public short  detectorOffsetOdd
         /// the process of expanding the polynomial.
         ///
         /// user-writable
+        ///
+        /// MZ: I think the current logic in the setter which invokes the 
+        ///     EEPROMChanged event has a big hole: namely, that the getter
+        ///     returns a reference to the entire array object.  That means
+        ///     the caller (customer software) can change the values of individual
+        ///     elements within the array at-will, and neither the setter nor
+        ///     event will ever fire.  I'm not sure that's entirely a bug,
+        ///     but wanted to document it.

```

Comment:
The EEPROMChanged event was always half baked and not particularly tested. Good note and something I'll look into when I have time

Reply:
In C++, I'd have the getter return a "const float[]", such that if the user wanted to change elements, they'd HAVE to use the setter to write an entire new array.  I haven't checked whether C# supports that or not.